cypress/webpack.config.js

```diff
@@ -41,4 +43,11 @@ module.exports = {
       },
     ],
   },
+  plugins: [
+    new webpack.DefinePlugin({
+      "process.env": {
+        IGNORE_TESTS: JSON.stringify(process.env.IGNORE_TESTS),
+      },
+    }),
+  ],
```

Comment:
Either way I'm pretty sure IGNORE_TESTS needed to exist and was causing issues, but will check for certain

Reply:
@nineteen88 yes sorry the suggestion is unclear there. No worries at this if its taking time to investigate, especially since Cypress will be removed soon anyway :)